content/en/blog/_posts/Use containerd to handle k8s.gcr.io deprecation

```diff
@@ -0,0 +1,174 @@
+
+This doesn’t mean that there’s no action required. Now is the time to scan code repositories and clusters for usage of the old registry. Failing to act will result in cluster components failing.
+
+Once the old registry goes away, **Kubernetes will not be able to create new Pods** (unless image is cached) if the container uses an image hosted on `k8s.gcr.io`.
+
+## What do you need to change?
+
+Cluster owners and development teams have to ensure they are not using any images stored in the old registry. The change is fairly simple. It's pretty simple, really.
+
+You need to change your manifests to use `registry.k8s.io` container registry.
+
+![Image.png](https://res.craft.do/user/full/9d54cc03-adfe-f72f-3389-565eb7356d1d/doc/13FE78CC-1DDD-47D9-B875-EE3FB15F916E/EB0981CC-3DC8-49EE-A6F5-A857A69CC949_2/D0F0mQ2JMAl55JNCuS66lUylRznVZywEMdojiRqkc0sz/Image.png)
+
+You can find out which Pods use the old registry using `kubectl`:
+
+```bash
+kubectl get pods --all-namespaces -o jsonpath="{.items[*].spec.containers[*].image}" |\0;31;37M0;31;38m
+tr -s '[[:space:]]' '\n' |\
+sort |\
+uniq -c | grep -i gcr.io
+```
+
+Here are the Pods in my test cluster that use the old registry:
+
+![Image.jpeg](https://res.craft.do/user/full/9d54cc03-adfe-f72f-3389-565eb7356d1d/doc/13FE78CC-1DDD-47D9-B875-EE3FB15F916E/ED8BD697-7BD9-4D58-83FC-833C66606E97_2/dcAQoH7ADPaoupccwi3WFMxt1PkS5ME4zXHHyxjAmdMz/Image.jpeg)
+
+These are the at-risk Pods. I’ll have to update the container registry used in the Pods.
+
+When hunting for references to old registry, be sure to include containers that may not be currently running in your cluster.
+
+## What if I don’t control the workloads?
+
+One of my colleagues raised an intriguing question. Is there’s a way to handle this change at a cluster level? He had a valid concern. Many large enterprises might not be able to implement this change in time before the community sunsets `k8s.gcr.io`.
+
+I work with many customers that manage large Kubernetes clusters, but have little control over the workloads that get deployed into the cluster. Some of these clusters are shared by hundreds of development teams. The burden is on Central Platform Engineering teams to dissipate this information to individual dev teams (who are busy writing code and not checking Kubernetes news!).
+
+So, what can these teams do to make sure when the old registry finally croaks, they don’t get paged for in the middle of the night for `ErrImagePull` and `ImagePullBackOff`errors?
+
+Turns out you can use containerd to handle this redirection at node level. Let’s find out how.
+
+## Using mirrors in containerd
+
+Ever since Dockerhub started rate limiting image pulls, many have opted to store images in local registries. Mirrors save network bandwidth, reduce image pull time, and don’t rate-limit.
```

Comment:
```suggestion
Ever since Docker Hub started rate limiting image pulls, many have opted to store images in local registries. Mirrors save network bandwidth, reduce image pull time, and don’t rate-limit.
```